vgstation13.dme

```diff
@@ -1516,9 +1516,9 @@
 #include "code\modules\components\ai\conversation.dm"
 #include "code\modules\components\ai\def_zone_handler.dm"
 #include "code\modules\components\ai\door_opener.dm"
+#include "code\modules\components\ai\hearing.dm"
```

Comment:
Please revert this change. It was presumably accidental and would probably cause problems with Git later.

Reply:
Fixed. Didn't even know the dme file was edittable.